prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="/contact-us/index.html">Click here to go to the home page</a>
```

Comment:
for this one you want to use the path `"../index.html"` to indicate that the browser should find the `index.html` file outside of the `contact-us` directory.